shell/rename-mysql4-class-to-resource.php

```diff
@@ -0,0 +1,573 @@
+<?php
+
+// DO NOT RUN DIRECTLY IN YOUR PRODUCTION ENVIRONMENT!
+// This script is distributed in the hope that it will be useful, but without any warranty.
+// See https://github.com/OpenMage/magento-lts/pull/2730 for more information
```

Comment:
I'd have the normal openmage license block, but actually I wouldn't have this file directly on the repo, because it's a one-use only. I'd maybe create a repo to be added to the dev-meta package or maybe even just a gist linked in the documentation?

Reply:
When this PR will be approved, there must be a repository (it would be best in OpenMage) where those interested can download and use it with their extensions. A README with warnings can also be created in that repository.

Reply:
Ok for the copyright notice, but this is not a one use script, you can reuse it each time you install a new module.

Reply:
ahh ok ok sorry, I thought it was for us and not really for the end users, my bad.